openwisp_notifications/api/views.py

```diff
@@ -198,11 +201,40 @@ def perform_create(self, serializer):
         )
 
 
+class OrganizationNotificationSettingView(GenericAPIView):
+    permission_classes = [IsAuthenticated, PreferencesPermission]
+    serializer_class = NotificationSettingUpdateSerializer
+
+    def post(self, request, user_id, organization_id):
+        notification_settings = NotificationSetting.objects.filter(
+            organization_id=organization_id, user_id=user_id
+        )
+        serializer = self.get_serializer(data=request.data)
+        if serializer.is_valid():
+            validated_data = serializer.validated_data
+            web = validated_data.get('web')
+            email = validated_data.get('email')
+
+            for notification_setting in notification_settings:
+                if web and not email:
+                    notification_setting.web = web
+                else:
+                    notification_setting.web = web
+                    notification_setting.email = email
+
+            NotificationSetting.objects.bulk_update(
+                notification_settings, ['web', 'email']
+            )
+            return Response(status=status.HTTP_200_OK)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
After another review, I came to conclusion that we don't need to make any changes to the serializer. Please make changes to the test for this endpoint as requested in my previous review. We will see if something breaks because this change.